client_test/liabilityswap_test.go

```diff
@@ -0,0 +1,177 @@
+package client_test
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/statechannels/go-nitro/channel/consensus_channel"
+	"github.com/statechannels/go-nitro/channel/state"
+	"github.com/statechannels/go-nitro/channel/state/outcome"
+	"github.com/statechannels/go-nitro/client"
+	"github.com/statechannels/go-nitro/client/engine/chainservice"
+	Token "github.com/statechannels/go-nitro/client/engine/chainservice/erc20"
+	"github.com/statechannels/go-nitro/client/engine/messageservice"
+	"github.com/statechannels/go-nitro/client/engine/store"
+	"github.com/statechannels/go-nitro/types"
+	"github.com/stretchr/testify/require"
+)
+
+var liabilitySwapChannelDeposit = big.NewInt(1000000000000000000) // 1 eth in wei
+
+// FIXME:
+// 1. Test fails during the ledger channel funding with this error:
+// 		0x111A00868581f73AB42FEEF67D235Ca09ca1E8db, error in run loop: failed to estimate gas needed: execution reverted: ERC20: transfer amount exceeds balance
```

Comment:
on certain testnet we need to set default maxGas as it's not specified in defaults

Reply:
this fixme is not about maxGas, it is about issue when using an ERC20 on outcome allocations

Reply:
issue ref: https://github.com/statechannels/go-nitro/issues/1001